srcs/ResponseHandler.cpp

```diff
@@ -244,25 +247,35 @@ void ResponseHandler::handlePut(request_s& request) {
 			this->_response += _status_codes[500];
 		}
 	}
-	_response += "\r\n";
+	handleCONTENT_LENGTH();
+	handleDATE();
+	handleCONTENT_TYPE(request);
+	handleCONNECTION_HEADER(request);
+
+	this->_response += "\r\n";
+	if (!_body.empty())
+		this->_response += _body + "\r\n";
```

Comment:
It's better to remove the "/r/n" after the body because it can cause an unsolicited response. The rfc doesn't specify the need for a CRLF after the body content.

Reply:
yep, good catch, thanks for fixing